src/network/graphql/queries.js

```diff
@@ -404,21 +403,22 @@ export async function allStats(client) {
 }
 
 /**
- * Queries most bets given the filters.
+ * Queries the event leaderboard entry based on event address and/or user address.
  * @param {ApolloClient} client Apollo Client instance.
  * @param {object} args Arguments for the query.
  * @return {object} Query result.
  */
-export async function mostBets(client, args) {
-  return new GraphQuery(client, QUERY_MOST_BETS, args).execute();
+export async function eventLeaderboardEntries(client, args) {
+  return new GraphQuery(client, QUERY_EVENT_LEADERBOARD_ENTRIES, args).execute();
 }
 
 /**
- * Queries the biggest winners based on an event address.
+ * Queries the global leaderboard entry based on a user address.
  * @param {ApolloClient} client Apollo Client instance.
  * @param {object} args Arguments for the query.
  * @return {object} Query result.
  */
-export async function biggestWinners(client, args) {
-  return new GraphQuery(client, QUERY_BIGGEST_WINNERS, args).execute();
+export async function globalLeaderboardEntries(client, args) {
+  const res = await new GraphQuery(client, QUERY_GLOBAL_LEADERBOARD_ENTRIES, args).execute();
```

Comment:
just return this. no reason to assign it first unless you were using `res`.

Reply:
oh right.... this was for testing, forgot to change back.